spec/features/Welcome/quotes_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Quotes Page' do
+  it "shows 0 results if no quotes were found" do
+    visit root_path
+    fill_in :keyword, with: 'jake'
+    click_on "Search for Quotes"
+    expect(page).to have_content('0 total results')
+  end
+
+  it "shows keyword used for search" do
+    visit root_path
+    fill_in :keyword, with: 'I am'
+    click_on "Search for Quotes"
+    expect(page).to have_content('for "I am"')
+  end
+
+  it "shows quotes and attributes" do
+    visit root_path
+    fill_in :keyword, with: 'I am'
+    click_on "Search for Quotes"
+    expect(page).to have_content('If I am not for myself, who will be for me?')
```

Comment:
For this test, you will probably want to use a `within block` or the `orderly gem` since you should be returning only the top 10. I would also include that test for a count. So example:
```
expect(page).to have_css('.quotes', count: 10)
within(first(".quotes")) do
   expect(page).to have_content("by: Rabbi Hillel")
   expect(page).to have_content("If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?")
   expect(page).to have_content("tags: famous-quotes, wisdom")
end
```
For the tags, we probably don't want them to be displayed in an array. So how could you make that happen?